src/index.ts

```diff
@@ -58,13 +31,19 @@ function fontawesomeSubset(subset: SubsetOption, outputDir: string, options: Fon
         light: "fa-light-300",
         regular: "fa-regular-400",
         brands: "fa-brands-400",
-        duotone: "fa-duotone-900"
+        duotone: "fa-duotone-900",
     };
     const fontTypes = Object.keys(fontMap);
 
+    let fontMeta = "";
+    let fontFiles = "";
+
     // Check to see if the user has either free, or pro installed.
-    if (!(existsSync("node_modules/@fortawesome/fontawesome-free") || existsSync("node_modules/@fortawesome/fontawesome-pro"))) {
-        console.error("Unable to find either the Free or Pro FontAwesome files in node_modules folder. Double-check that you have your preferred fontawesome package as a dependency in `package.json` and rerun the installation.");
+    if (existsSync(`node_modules/@fortawesome/fontawesome-${options.package}`)) {
```

Comment:
Is there a better way to get the path to the fontawesome installed files? Not all projects use a local `node_modules` install. In my original script I had used `require.resolve` (although I have no idea if that is proper, I'm no JS expert).

Reply:
Thanks for pointing that out, I just checked around on stack overflow and they also recommend how you have it. I'll update the pr. Thanks for the code review, I really appreciate that.

If you don't have fa installed, this error will popup. The free/pro is dynamic still:
![image](https://user-images.githubusercontent.com/17788706/175612520-6db7c1d8-36dd-48dd-a05d-e06065153b13.png)